azure-pipelines.yml

```diff
@@ -9,7 +9,6 @@ trigger:
     include:
     - master
     - release/*
-    - internal/release/*
```

Comment:
Instead make these conditional on the queue e.g. something like
``` yaml
${{ if ne(variables['System.TeamProject'], 'internal') }}:
  - release/*
${{ if eq(variables['System.TeamProject'], 'internal') }}:
  - internal/release/*
```

Reply:
We should not build internal/release/* at all right now. Arcade publishing is not designed to handle this and we do not want to accidentally push internal-only fixes publicly. I'm sending mail about this and will CC you